### PR TITLE
Implemented assertEqualsToIgnoringNewLines

### PR DESCRIPTION
**Pull request by dafenix**
#### Check List:
* Fixes Implemented assertEqualsToIgnoringNewLines
* Unit tests : YES
* Javadoc with a code example (API only) : YES

### CODE REVIEW

**Comment by joel-costigliola** [comment]
Thanks, I will review your PR in the next days !

**Comment by joel-costigliola** [comment]
You have forgotten to add `isEqualToIgnoringNewLines` in `AbstractCharSequenceAssert` ;-)

File: src/main/java/org/assertj/core/error/ShouldBeEqualIgnoringNewLines.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package org.assertj.core.error;
+
+public class ShouldBeEqualIgnoringNewLines  extends BasicErrorMessageFactory {
+	 public static ErrorMessageFactory shouldBeEqualIgnoringNewLines(CharSequence actual, CharSequence expected) {
+	    return new ShouldBeEqualIgnoringNewLines(actual, expected);
+	 }
+
+	  private ShouldBeEqualIgnoringNewLines(CharSequence actual, CharSequence expected) {
+	    super("%nExpecting:%n  <%s>%nto be equal to:%n  <%s>%nignoring newline (\n, \r\n) in acutal.",
```
Comment (joel-costigliola) [inline src/main/java/org/assertj/core/error/ShouldBeEqualIgnoringNewLines.java]: typos: `newline` -> `newlines`, `acutal` -> `actual`

Comment (dafenix) [reply src/main/java/org/assertj/core/error/ShouldBeEqualIgnoringNewLines.java]: typos fixed

**Comment by joel-costigliola** [comment]
Integrated after minor updates (mostly tests) thanks @dafenix!

**Pull request closed without merging**